demo/api.min.js

```diff
@@ -0,0 +1,29 @@
+function toggledTextExample() {
+  const addEventListener = (element) => {
+    element.addEventListener('click', () => {
+      element.dispatchEvent(new CustomEvent('auroButton-toggledText'));
+    });
+  };
+
+  const toggledTextExampleElemLeft = document.getElementById('toggledTextExampleLeft');
+  const toggledTextExampleElemRight = document.getElementById('toggledTextExampleRight');
+
+  addEventListener(toggledTextExampleElemLeft);
+  addEventListener(toggledTextExampleElemRight);
+}
+
+function initButtonApiExamples(initCount) {
+  initCount = initCount || 0;
+
+  try {
```

Comment:
is there some event or something we can listen to instead of re-attempting the event listener addition multiple times?

Reply:
That is a great question and something I can bring to Jason's attention.

Reply:
If we just want to wait for the content to load some googling is showing we should be able to use the [window on load event](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event)? Might be worth trying out?

Reply:
I think the main issue is that we need to wait for all of the html to load before we run the javascript. I don't think everything loads right when the window does.

@jason-capsule42 What do you think of Isaac's comment/idea?

Reply:
@Izook this is great idea to explore further. However, I'd like to get the button release out using the pattern we currently have and then let's dive deeper into how this could work for all auro-components as a separate ticket. Could you open a new ticket in the generator for this work?